analyses/cell-type-ewings/README.md

```diff
@@ -133,14 +145,23 @@ This table will be saved in `references/cell_lists/<sample_id>/<library_id>_refe
 | `cellassign_celltype_annotation` | Original annotation as obtained by `CellAssign` in the processed `SingleCellExperiment` object |
 | `singler_celltype_annotation` | Original annotation as obtained by `SingleR` in the processed `SingleCellExperiment` object |
 
-## Cell type annotations
+### Computational resources
+
+Currently, the `cnv-annotation.sh` can only be run using 1 CPU if running locally.
+Use `threads=1` when running the workflow to specify this.
+
+To increase the speed of the workflow, we recommend running on a computer with at least 12 CPUs.
+If this is the case, you may run the workflow using the default settings of 4 threads.
```

Comment:
I don't think I understand this? If I am running locally, why can't I use more than one CPU?

Reply:
I can't seem to find the PR where we brought this up originally, but both of us were having trouble running with multiple CPUs and CellAssign failed with a seg fault, but with 1 CPU it worked.

Reply:
I think maybe we should clarify then that this may fail on a Mac, specifically.